sleap/gui/commands.py

```diff
@@ -3750,6 +3751,67 @@
 
         return views
 
+    @staticmethod
+    def get_permutations_of_instances(
+        selected_instance: Instance,
+        session: RecordingSession,
+        frame_idx: int,
+        cams_to_include: Optional[List[Camcorder]] = None,
+    ) -> Dict[int, Dict[Camcorder, List[Instance]]]:
+        """Get all (single-instance) possible combinations of instances across views.
+
+        Args:
+            selected_instance: The `Instance` to add to  permutations of instances in
+                views other than that of the `selected_instance`.
+            session: The `RecordingSession` containing the `Camcorder`s.
+            frame_idx: Frame index to get instances from (0-indexed).
+            cams_to_include: List of `Camcorder`s to include. Default is all.
+            require_multiple_views: If True, then raise and error if one or less views
+                or instances are found.
+
+        Raises:
+            ValueError if one or less views or instances are found.
+
+        Returns:
+            Dict with frame identifier keys (not the frame index) and values of another
+            inner dict with `Camcorder` keys and `List[Instance]` values. Each
+            `List[Instance]` is of length 1.
+        """
+
+        cam_selected = session.get_camera(selected_instance.video)
+        cam_selected = cast(Camcorder, cam_selected)  # Could be None if not in session
+
+        # Get all instances accross views at this frame index, then remove selected
+        instances_excluding_selected: Dict[
+            Camcorder, List[Instance]
+        ] = TriangulateSession.get_instances_across_views(
+            session=session,
+            frame_idx=frame_idx,
+            cams_to_include=cams_to_include,
+            track=-1,  # Get all instances regardless of track.
+            require_multiple_views=True,
+        )
+        instances_excluding_selected.pop(cam_selected)
+
```

Comment:
I actually want this to raise an error since it would be incorrect if the `selected_instance`'s `Camcorder` was not in the initial `instances_excluding_selected`